master/getting-started/kubernetes/upgrade/delete.md

```diff
@@ -3,5 +3,68 @@ title: Deleting old data
 no_canonical: true
 ---
 
-_Docs on this coming soon!_
+## About deleting old Calico data
 
+You may need or wish to manually delete Calico data from your etcd datastore under the
+following conditions.
+
+- [A data migration attempt failed partway through, leaving the etcdv3 datastore
+  with some, but not all of your etcvd2 data](#deleting-calico-data-from-etcdv3-after-a-partial-migration).
+  
+- [You succeeded in migrating your data and upgrading to Calico v3.0. After
+  running Calico for some time and experiencing no errors, you want to delete
+  the Calico data from the etcdv2 datastore](#deleting-calico-data-from-etcdv2-after-a-successful-migration-and-upgrade).
+  
+
+## Deleting Calico data from etcdv3 after a partial migration
+
+1. Obtain the name of the etcd pod. 
+
+   ```
+   kubectl get pods --all-namespaces
+   ```
+
+1. Copy the name of the etcd pod.
```

Comment:
There are two etcd related pods for a kubeadm installed cluster. One is for kubernetes and one is for calico. I suggest we specify the one with `calico-etcd` prefix.
```kube-system   calico-etcd-xv45j                          1/1       Running            0          16m
kube-system   calico-kube-controllers-755c68d654-glchb   1/1       Running            0          16m
kube-system   calico-node-9vmng                          1/2       Running            4          16m
kube-system   calico-node-mlj2q                          1/2       Running            4          16m
kube-system   calico-node-rflvk                          1/2       Running            3          16m
kube-system   etcd-song-master                           1/1       Running            4          6d
kube-system   kube-apiserver-song-master                 1/1       Running            4          6d
kube-system   kube-controller-manager-song-master        1/1       Running            1          23h```

Reply:
I think we need to drop this part of the guide, since it's specific to kubeadm and not going to be right for a large number of users. See above comment.